php/resources/root/docker-entrypoint.sh

```diff
@@ -4,55 +4,22 @@ set -e
 # exit when docker sends SIGTERM (aka docker stop)
 onTerminate() {
     echo "Got SIGTERM, stopping apache..."
-    kill `cat /run/apache2/apache2.pid`
+    kill `cat /var/run/httpd/httpd.pid`
     exit
 }
 trap 'onTerminate' SIGTERM
 
-#
-# Ensure database is set up correctly
-#
-
-# Wait for database to be started...
-until echo "show tables" | mysql -h db -u limsdbuser --password='limsdbpassword' limsdockerdb &> /dev/null
-do
-    echo "Waiting for mysql..."
-    sleep 2
-done
-
-mysql -u root --password='root' -h db < /var/lims/setup/legacy-setup.sql
-mysql -u root --password='root' -h db < /var/lims/setup/setup.sql
-
-#
-# Initialize container-specific files
-#
-
 # Ensure limsuser uid and gid matches environment configuration (see docker-compose.yml)
 : ${DOCKERLIMS_SHELL_UID:=5000}
 echo "Switching to UID ${DOCKERLIMS_SHELL_UID}"
 usermod -u ${DOCKERLIMS_SHELL_UID} limsuser
 groupmod -g ${DOCKERLIMS_SHELL_UID} limsuser
 
-
 # Ensure limsuser will have permissions on all lims-related files
 chown -R limsuser:limsuser /var/lims/
 
-# This will install dependencies. --no-scripts is necessary because building
```

Comment:
This section is removed because these can be run from command-line. 

Thinking ahead, if we were to use Kubernetes or Swarm, we should be able to spin up/down any number of these containers or respawn if one dies. Or, as I hope, this container can be used for local dev. By including these scripts, we are forcing these scripts to run each time the container gets spun up.

Reply:
Talked with Eric and we decided to keep the autosetup code in the entrypoint but add a check for whether the database host exists. If it doesn't, all this logic is skipped.

Implemented in 03f661b222f0ef059a702163595f268cc0e07869